tools/cli/internal/changelog/outputfilter/outputfilter.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const lan = "en" // language for localized output
 
-type Entry struct {
+type OasDiffEntry struct {
```

Comment:
[q] any benefit in using https://github.com/Tufin/oasdiff/blob/170218fb7fb9181aff79b64e4981e661990c9cf7/checker/api_change.go#L17?

Reply:
note - if any benefit, not in this PR anyways

Reply:
Are you proposing to use `ApiChange` instead of `OasDiffEntry` or to add  `ApiChange` inside `OasDiffEntry`?

Reply:
Oasdiff entry could be composed by ApiChange  + our fields since we still need HideFromChangelog - but only if we see any benefit